docs/backends/bigquery.qmd

```diff
@@ -77,17 +77,21 @@ con = ibis.bigquery.connect(
 
 ::: {.callout-note}
 
-`ibis.bigquery.connect` is a thin wrapper around [`ibis.backends.bigquery.Backend.do_connect`][ibis.backends.bigquery.Backend.do_connect].
+`ibis.bigquery.connect` is a thin wrapper around [`ibis.backends.bigquery.Backend.do_connect`](ibis.backends.bigquery.Backend.do_connect).
 
 :::
 
 ### Connection Parameters
 
-<!-- prettier-ignore-start -->
-::: ibis.backends.bigquery.Backend.do_connect
-    options:
-      heading_level: 4
-<!-- prettier-ignore-end -->
+```{python}
+#| echo: false
+#| output: asis
+from quartodoc import get_object, MdRenderer
+
+renderer = MdRenderer(header_level=4)
+obj = get_object("ibis.backends.bigquery", "Backend")
+print(renderer.render(obj.get_member("do_connect")))
```

Comment:
Ok, my desire to avoid copypasta got the better of me, there's now a `_utils.py` module that wraps up a few of the repetitive bits into a few functions.